aiplatform/src/main/java/aiplatform/imagen/GenerateImageSample.java

```diff
@@ -0,0 +1,108 @@
+import com.google.protobuf.Value;
+import com.google.protobuf.util.JsonFormat;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.Base64;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
+public class GenerateImageSample {
+
+  public static void main(String[] args) throws IOException {
+    // TODO(developer): Replace these variables before running the sample.
+    String projectId = "my-project-id";
+    String location = "us-central1";
+    String prompt = ""; // The text prompt describing what you want to see.
+
+    generateImage(projectId, location, prompt);
+  }
+
+  static PredictResponse generateImage(String projectId, String location, String prompt)
+      throws IOException {
+    final String endpoint = String.format("%s-aiplatform.googleapis.com:443", location);
+    PredictionServiceSettings predictionServiceSettings =
+        PredictionServiceSettings.newBuilder().setEndpoint(endpoint).build();
+
+    // Initialize client that will be used to send requests. This client only needs to be created
+    // once, and can be reused for multiple requests.
+    try (PredictionServiceClient predictionServiceClient =
+        PredictionServiceClient.create(predictionServiceSettings)) {
+
+      final EndpointName endpointName =
+          EndpointName.ofProjectLocationPublisherModelName(
+              projectId, location, "google", "imagegeneration@005");
+
+      Map<String, Object> instancesMap = new HashMap<>();
+      instancesMap.put("prompt", prompt);
+      Value instances = mapToValue(instancesMap);
+
+      Map<String, Object> paramsMap = new HashMap<>();
+      paramsMap.put("sampleCount", 1);
+      // You can't use a seed value and watermark at the same time.
+      // paramsMap.put("seed", 100);
+      // paramsMap.put("addWatermark", true);
+      paramsMap.put("aspectRatio", "1:1");
+      paramsMap.put("safetyFilterLevel", "block_some");
+      paramsMap.put("personGeneration", "allow_adult");
+      Value parameters = mapToValue(paramsMap);
+
+      PredictResponse predictResponse =
+          predictionServiceClient.predict(
+              endpointName, Collections.singletonList(instances), parameters);
+
+      for (Value prediction : predictResponse.getPredictionsList()) {
+        Map<String, Value> fieldsMap = prediction.getStructValue().getFieldsMap();
+        if (fieldsMap.containsKey("bytesBase64Encoded")) {
+          String bytesBase64Encoded = fieldsMap.get("bytesBase64Encoded").getStringValue();
+          Path tmpPath = Files.createTempFile("imagen-", ".png");
+          Files.write(tmpPath, Base64.getDecoder().decode(bytesBase64Encoded));
+          System.out.format("Image file written to: %s\n", tmpPath.toUri());
+        }
+      }
+      return predictResponse;
+
+    } catch (ApiException e) {
+      System.out.format("Exception: %s\n", e.getLocalizedMessage());
+    }
```

Comment:
Don't catch the exception unless we show how to handle it. Kindly throw the exception.

Reply:
Done